Content.Shared/Traits/TraitPrototype.cs

```diff
@@ -60,4 +61,12 @@ public sealed partial class TraitPrototype : IPrototype
     /// </summary>
     [DataField]
     public ProtoId<TraitCategoryPrototype>? Category;
+
+    // Begin DeltaV Additions - Species trait hiding
+    /// <summary>
+    /// Hides traits from specific species
+    /// </summary>
+    [DataField]
+    public HashSet<ProtoId<SpeciesPrototype>> ExcludedSpecies = new();
+    // End DeltaV Additions
```

Comment:
still needs changing to just be `<summary>DeltaV - ...</summary>` and no begin/end